sdk/cpp/core/src/gnmi_client.cpp

```diff
@@ -0,0 +1,273 @@
+	int element_size;
+	std::string prefix_to_prepend;
+	std::string path_to_prepend;
+	int notification_size = response->notification_size();
+
+	// read from all response fields and create reply
+	for(int i = 0; i < notification_size; ++i) 
+	{
+		notification = response->notification(i);
+
+		if(notification.has_prefix()) {
+			prefix = notification.prefix();
+			element_size = prefix.element_size();
+			for(int j = 0; j < element_size; ++j)
+            {
+              	element.append("\"");
+		        element.append(prefix.element(j));
+		        element.append("\" ");
+		        prefix_to_prepend.append(prefix.element(j));
+		        if(j != (element_size - 1))
+		        {
+		        	prefix_to_prepend.append(":");
+		        }
+            }
+            prefix_to_prepend.append("\": {");
+		}
+
+		int update_size = notification.update_size();
+		
+		if(notification.update_size() != 0) {
+	        for(int k = 0; k < update_size; ++k) 
+	        {
+	            update = notification.update(k);
+	            
+	            if(update.has_path()) 
+	            {
+	            	path = update.path();
+	           		element_size = path.element_size();
+	             	element.clear();
+	              	for(int l = 0; l < element_size; ++l) 
+	              	{
+		                element.append("\"");
+		                element.append(path.element(l));
+		                element.append("\" ");
+		                path_to_prepend.append(path.element(l));
+		                if(l != element_size-1)
+		                {
+		                	path_to_prepend.append(":");
+		                }
+	              	}
+	              path_to_prepend.append("\": {");
+	            }
+
+	            if(update.has_value()) 
+	            {
+	              	value = update.value();
+	              	reply_to_parse.append("\"data\": {\"");
+	              	reply_to_parse.append(prefix_to_prepend); 
+	              	reply_to_parse.append(value.value());
+	              	reply_to_parse.append(" } } }");
+            	}
+           	}   
+        }
+	}
+	return reply_to_parse;
+}
+
+std::string gNMIClient::execute_wrapper(const string & payload)
+{
+	GetRequest request;
+	GetResponse response;
+	request = gNMIClient::ParseGetRequest(request, payload);
+	std::cout << "===============Get Request Sent================" << std::endl;
+	std::cout << request.DebugString() << std::endl;
+	std::string reply = execute_payload(payload, request, &response);
+	return reply;
+}
+
+std::string gNMIClient::execute_payload(const string & payload, const GetRequest& request, GetResponse* response)
+{
+	grpc::ClientContext context;
+	grpc::Status status;
+
+	status = stub_->Get(&context, request, response);
+	std::cout << "=============Get Response Received=============" << std::endl;
+	std::cout << response->DebugString() << std::endl;
+
+	if (!status.ok()) 
+	{
+		return status.error_message();
+	} 
+	else 
+	{
+		std::cout << "===================RPC Passed===================" << std::endl;
```

Comment:
Could we use `YLOG_XXX` as you did for gnmi provider? Ignore this comment if stdout is preferred for some reason.